go/store/nbs/store_test.go

```diff
@@ -263,7 +263,7 @@ func TestNBSCopyGC(t *testing.T) {
 	wg := &sync.WaitGroup{}
 	wg.Add(1)
 	go func() {
-		msErr = st.MarkAndSweepChunks(ctx, r, keepChan)
+		msErr = st.MarkAndSweepChunks(ctx, r, keepChan, nil)
```

Comment:
```suggestion
		msErr = st.MarkAndSweepChunks(ctx, r, keepChan, st)
```

Reply:
seems like it shouldn't be nil?

Reply:
?  it's not nil.

Reply:
the last argument to `MarkAndSweepChunks` is `nil`